### DISCUSSION TITLE
ambiguous qualifier 'd' (debug,die)

### DISCUSSION THREAD

**Discussion by karim-jpg** [projectdiscovery/chaos-client#89] [Q&A] [answered]
hello I have this issue

└─$ chaos -d uber.com 1 ⨯
Warning: ambiguous qualifier 'd' (debug,die)
Died: Unknown qualifier -d

**Comment by ehsandeep** [comment, marked as answer]
@karim-jpg this doesn’t look like an error from chaos client project, could you confirm if you installed the correct binary?

**Reply by MetzinAround** [reply]
Feel free to re-open if you can provide more info!

**Reply by MetzinAround** [reply]
Feel free to re-open if you can provide more info!
